src/migrations/20220317164432-create-event-relationships.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('EventRelationships', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userFromId: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        unique: true,
+        // references: {
```

Comment:
Why does these references are merged?
That's because someone else work with this model?
If so, please create PR to uncomment it

Reply:
I think it would be better to leave it at that. When we unite everything, it will be possible to unpack everything step by step